### ISSUE TITLE
Sometimes it display different title, artist and image.

### ISSUE THREAD

**Issue by Bossfrosty** [jailsonsb2/RadioplayerAPI#1]
Sometimes, the currently playing song information from Zeno doesn't match the actual artist and song title, but if you check the history / api, it displays/saved the correct information. Just wondering why sometimes it gets different title.

**Comment by jailsonsb2** [comment]
![image](https://github.com/jailsonsb2/RadioplayerAPI/assets/13502945/863e6b1c-7786-426c-84dc-4fd487c1e193)
In some API responses, it was receiving the value "undefined" and performing the album cover search using this term. I implemented a check to prevent it from searching for the album cover if the value is "undefined," thus displaying the default cover instead.

**Comment by jailsonsb2** [comment]
Just update the main.js file with the new available version.

**Comment by Bossfrosty** [comment]
Haven't tried your updated main.js, but I tried changing

currentStation.server || "itunes" to currentStation.server || "spotify"

yesterday. I think it's working perfectly now. I'm not sure what else could happen after I changed it to "spotify."

**Comment by jailsonsb2** [comment]
You can pass this parameter via index file -> <script> highlighted in the photo; If left empty = itunes. If you put spotify, it will search for spotify.

![image](https://github.com/jailsonsb2/RadioplayerAPI/assets/13502945/e1d77efd-58ab-43f1-9f38-b6a77e297fbd)

**Comment by jailsonsb2** [comment]
I value your feedback and am actively working to refine the instructions. While some details like this were initially overlooked, I'm incorporating your suggestions to ensure clarity and comprehensiveness.

**Comment by jailsonsb2** [comment]
![image](https://github.com/jailsonsb2/RadioplayerAPI/assets/13502945/de1a80a6-3cd7-4dde-94ee-dd0cd2076664)


You were absolutely right! There was a detail in the iTunes album cover search function. It had an option, now commented out in the code, that would replace the original song and artist names with the information returned by the iTunes API. This was the reason why the names were sometimes different.

In other words, the function was correctly finding the album cover, but when it used the iTunes API data to fill in the music metadata, it would overwrite the original names. With the option commented out, the function now keeps the original song and artist names, ensuring data consistency.
